src/Uno.UI/UI/Xaml/Controls/Slider/Slider.Uno.cs

```diff
@@ -29,6 +33,10 @@ private void Slider_Loaded(object sender, RoutedEventArgs e)
 		{
 			AttachHorizontalThumbSubscriptions();
 		}
+
+#if __WASM__
```

Comment:
Please don't do that per control. This has to be the default behavior. User doesn't expect that content that is not flagged as draggable in the app to be in some browsers.